gulpfile.js

```diff
@@ -317,22 +309,22 @@ gulp.task('build', gulp.series(
 
 gulp.task('default', gulp.series('build'));
 
-gulp.task('watch', gulp.series('clean:before', 'copy:theme', async () => {
+gulp.task('watch', gulp.series(['clean:before', 'copy:theme', 'precompile', 'move:docimage', 'move:images', 'useref', 'move:static', 'move:helpers', 'clean:after', 'revfiles', 'revreplace:content', 'revreplace:theme'], async () => {
```

Comment:
The `async` function is not inside the `gulp.series` array while later in line 318 it is. Is this expected?

Reply:
It was a mistake. I also fix another error.